client/classic/upe/payment-processing.js

```diff
@@ -161,6 +162,16 @@ async function createStripePaymentElement( api, paymentMethodType ) {
 		}
 	}
 
+	const setupFutureUsage =
+		document.getElementById( 'wc-stripe-new-payment-method' ).checked ||
+		getStripeServerData()?.cartContainsSubscription;
+	if ( setupFutureUsage ) {
+		options = {
+			...options,
+			setupFutureUsage: 'off_session',
+		};
+	}
+
```

Comment:
By placing this here, we're also changing the options setup for non-OC. Is this intentional?

Reply:
Before the OC implementation, the shortcode checkout had no `setupFutureUsage` option set. This could potentially fix other issues with non-OC transactions. But I will bind it to OC for now to avoid unintended changes and keep the scope of the PR.